Power Source negative capacity

Test failure on Linux on aarch64 (M1):
```
[ERROR] Failures (1):
[ERROR]   JUnit Jupiter:PowerSourceTest:testPowerSource()
[ERROR]     MethodSource [className = 'oshi.hardware.PowerSourceTest', methodName = 'testPowerSource', methodParameterTypes = '']
[ERROR]     => java.lang.AssertionError: Power Source's remaining capacity shouldn't be negative
[ERROR] Expected: is a value equal to or greater than <0.0>
[ERROR]      but: <-1.0> was less than <0.0>
[ERROR]        org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
[ERROR]        oshi.hardware.PowerSourceTest.testPowerSource(PowerSourceTest.java:51)
```
Stats:
```
ERROR] Power Sources: 
[ERROR]  Name: tps6598x-source-psy-0-0038, Device Name: unknown,
[ERROR]  RemainingCapacityPercent: -100.0%, Time Remaining: Unknown, Time Remaining Instant: Unknown,
[ERROR]  Power Usage Rate: 0.0mW, Voltage: unknownAmperage: 0.0mA,
[ERROR]  Power OnLine: false, Charging: false, Discharging: false,
[ERROR]  Capacity Units: RELATIVE, Current Capacity: -1, Max Capacity: 1, Design Capacity: 1,
[ERROR]  Cycle Count: -1, Chemistry: unknown, Manufacture Date: unknown, Manufacturer: unknown,
[ERROR]  SerialNumber: unknown, Temperature: unknown
[ERROR]  Name: tps6598x-source-psy-0-003f, Device Name: unknown,
[ERROR]  RemainingCapacityPercent: -100.0%, Time Remaining: Unknown, Time Remaining Instant: Unknown,
[ERROR]  Power Usage Rate: 0.0mW, Voltage: unknownAmperage: 0.0mA,
[ERROR]  Power OnLine: false, Charging: false, Discharging: false,
[ERROR]  Capacity Units: RELATIVE, Current Capacity: -1, Max Capacity: 1, Design Capacity: 1,
[ERROR]  Cycle Count: -1, Chemistry: unknown, Manufacture Date: unknown, Manufacturer: unknown,
[ERROR]  SerialNumber: unknown, Temperature: unknown
```

--- a/oshi-core/src/main/java/oshi/hardware/common/AbstractPowerSource.java
+++ b/oshi-core/src/main/java/oshi/hardware/common/AbstractPowerSource.java
@@ -266,7 +266,7 @@ public abstract class AbstractPowerSource implements PowerSource {
         if (getVoltage() > 0) {
             sb.append(getVoltage()).append("V, ");
         } else {
-            sb.append(Constants.UNKNOWN);
+            sb.append(Constants.UNKNOWN).append(", ");
         }
         sb.append("Amperage: ").append(getAmperage()).append("mA,\n ");
         sb.append("Power OnLine: ").append(isPowerOnLine()).append(", ");

--- a/oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxPowerSource.java
+++ b/oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxPowerSource.java
@@ -100,8 +100,9 @@ public final class LinuxPowerSource extends AbstractPowerSource {
                         UdevDevice device = udev.deviceNewFromSyspath(syspath);
                         if (device != null) {
                             try {
-                                if (ParseUtil.parseIntOrDefault(device.getPropertyValue("POWER_SUPPLY_PRESENT"),
-                                        1) > 0) {
+                                if (ParseUtil.parseIntOrDefault(device.getPropertyValue("POWER_SUPPLY_PRESENT"), 1) > 0
+                                        && ParseUtil.parseIntOrDefault(device.getPropertyValue("POWER_SUPPLY_ONLINE"),
+                                                1) > 0) {
                                     psName = getOrDefault(device, "POWER_SUPPLY_NAME", name);
                                     String status = device.getPropertyValue("POWER_SUPPLY_STATUS");
                                     psCharging = "Charging".equals(status);

